test/data/confirmations/personal_sign.ts

```diff
@@ -9,6 +9,7 @@ export const unapprovedPersonalSignMsg = {
   time: new Date().getTime(),
   type: 'personal_sign',
   securityProviderResponse: null,
+  chainid: '0x5',
```

Comment:
Should this also use `mockState.metamask.networkConfigurations.goerli.chainId`?

Reply:
The object not not using `mockState.metamask` otherwise thus I would avoid importing it for just chainId.